shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -1,7 +1,7 @@
 <template name="loginButtons">
   {{!-- Data context is an instance of AccountsUi. --}}
   {{#if isDemoUser}}
-    <a class="{{#if expiringSoon}}expiring-soon{{/if}}">Demo {{demoTimeLeft}} ▾</a>
+    <a>Create Account ▾</a>
```

Comment:
I actually have come to the realization that I vastly prefer "create account" over "sign up".

"Sign up" sounds like I'm committing to something, or paying for something. That's scary, and it's why I always resisted saying "sign up" rather than "sign in" in the past. "Create account" sounds much more passive: it's just creating a bucket to put my data in.